pkg/network/state.go

```diff
@@ -93,12 +93,16 @@ type telemetry struct {
 
 const minClosedCapacity = 1024
 
+type closedKey string
```

Comment:
Not sure we need this anymore; I'd be ok with removing it and using `string`.

Reply:
👍